Resources/Prototypes/Catalog/Cargo/cargo_engines.yml

```diff
@@ -4,7 +4,7 @@
     sprite: Objects/Power/AME/ame_jar.rsi
     state: jar
   product: CrateEngineeringAMEJar
-  cost: 2000
+  cost: 20000
```

Comment:
needs DeltaV comment + 20k is too extreme and solely relies on top 1% cargo techs speedrunning bounties or mining nonstop, should be 5k or so imo